src/btree/bt_delete.c

```diff
@@ -18,9 +18,9 @@
  * pages of the truncate range, then walks the tree with a flag so the tree walk code skips reading
  * eligible pages within the range and instead just marks them as deleted, by changing their WT_REF
  * state to WT_REF_DELETED. Pages ineligible for this fast path include pages already in the cache,
- * having overflow items, or requiring lookaside records. Ineligible pages are read and have their
- * rows updated/deleted individually. The transaction for the delete operation is stored in memory
- * referenced by the WT_REF.page_del field.
+ * having overflow items, or requiring history store records. Ineligible pages are read and have
```

Comment:
There are many places also mentioning cache overflow. I'm not sure we still can call the history store a cache overflow mechanism.

Reply:
I would suggest doing that as part of WT-5454, I'll update the ticket to also update comments that refer to the cache overflow.